components/header/header.tsx

```diff
@@ -1,7 +1,10 @@
 import './header-style.scss';
 
 function Header () {
-    return <h1 className="title">Tutorials and tips</h1>
+    return <div className="header">
+        <h1 className="header--title">Tutorials and tips</h1>
+        <p className="header--slogan">to become a great web developer</p>
+    </div>
```

Comment:
Her ser jeg du bruker BEM - block element modifier - for å strukturere CSSen din. Det er kjempebra! Men her tror jeg du har blandet elements (de som ser `__slik-ut`) og modifiers (de som ser `--slik-ut`). 

Her er `header` en block, og både `title` og `slogan` er "elementer" inni denne blokka. Hadde du derimot hatt to forskjellige versjoner av en title - en rød og en blå, så kunne du skrevet `header__title header__title--red` for å spesifisere det.

Ga det mening? :)